integration_tests/e2e/goal/createGoal.cy.ts

```diff
@@ -24,22 +24,45 @@ context('Create a goal', () => {
     page.isForPrisoner(prisonNumber)
   })
 
+  it('should not proceed to add step page given validation errors on create goal page', () => {
+    // Given
+    const prisonNumber = 'G6115VJ'
+    cy.signIn()
+    cy.visit(`/plan/${prisonNumber}/goals/create`)
+
+    const page = Page.verifyOnPage(CreateGoalPage)
+    page //
```

Comment:
Minor "tidying" ... seeing as the methods all return "this" we can chain them .... but unfortunately prettier feels we can and should put it all on one line (which makes it harder to read IMHO), so we have to trick prettier by adding the comment slashes !

Reply:
Ok, but I'm not sure if the chaining adds any value here 🤔

Reply:
Thats a fair challenge. I can put these back to how they were if we like, but I think in subsequent tests where we might want to assert several things on the page the chaining will come into its own 👍 
Shall I put these ones back for now, on the basis that we dont really need the chaining syntax here and now?

Reply:
Nah, I'd leave it now you've done it 👍

Reply:
Cool, will do 👍